app/routes/meetings/components/PizzaAccordion.tsx

```diff
@@ -51,13 +51,19 @@ export const PizzaAccordion = ({ meeting, meetingInvitations }: Props) => {
       >
         {
           <div>
-            <h3>Skal ha pizza</h3>
+            <Flex alignItems="baseline">
+              <h3>Skal ha pizza</h3>
+              <p className={styles.pizzaCount}>({pizzaUsers.length})</p>
```

Comment:
```suggestion
              <span className={styles.pizzaCount}>({pizzaUsers.length})</span>
```
Very picky, but I think it's good practice to not use paragraph tags for thing that are not paragraphs considering that this comes with default styling